modules/cnv-creating-vm-wizard-web.adoc

```diff
@@ -22,15 +22,14 @@ A *Bootable Disk* is not required for virtual machines provisioned from a *PXE*
 .Procedure
 
 . Click *Workloads* -> *Virtual Machines* from the side menu.
-. Click *Create Virtual Machine* and select *Create with Wizard*.
-. Fill in all required *Basic Settings*. Selecting a *Template* automatically fills in these fields.
-. Click *Next* to progress to the *Networking* screen. A `nic0` NIC is attached by default.
-.. (Optional) Click *Create NIC* to create additional NICs.
-.. (Optional) You can remove any or all NICs by clicking the &#8942; button and selecting *Remove NIC*. A virtual machine does not need a NIC attached to be created. NICs can be created after the virtual machine has been created.
+. Click *Create Virtual Machine* and select *New with the Wizard*.
```

Comment:
Sorry, I made a mistake in the mail. It's "New with Wizard".

Reply:
Made the change to "New with Wizard"